### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getEntries' of null

### ISSUE THREAD

**Issue by sadlerdavid** [tualo/tualo-git-context#7]
Uncaught TypeError: Cannot read property 'getEntries' of null

**Atom Version**: 0.158.0
**System**: Mac OS X 10.10.1
**Thrown From**: [tualo-git-context](https://github.com/tualo/tualo-git-context) package, v0.3.0
### Steps To Reproduce
1. ...Launching Atom
2. ...
### Stack Trace

```
At /Users/DavidSadler/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:191

TypeError: Cannot read property 'getEntries' of null
  at module.exports.TualoGitContextView.refreshTree (/Users/DavidSadler/.atom/packages/tualo-git-context/lib/tualo-git-context-view.coffee:191:9)

```

/cc @atom/core

**Comment by fotanus** [comment]
![true story](http://i1.kym-cdn.com/photos/images/facebook/000/141/644/7nTnr.png)

**Comment by thomashoffmann1979** [comment]
Thanks a lot for filing this issue, i have fixed it in version 0.5.0.
